src/darker/linting.py

```diff
@@ -39,7 +39,13 @@ def _parse_linter_line(line: str, root: Path) -> Tuple[Path, int, str, str]:
     #       description = "error: Foo"
     try:
         location, description = line[:-1].split(": ", 1)
```

Comment:
```suggestion
        location, description = line.strip().split(": ", 1)
```
Wouldn't stripping be better to get rid of newlines?

Reply:
`.strip()` gets rid of whitespace at the beginning of the line as well. I would prefer to be strict in parsing and only consider unindented lines. I can imagine some linter outputting supplementary information on indented lines, and if they just happen to be formatted like e.g.
```bash
······<something>··:··<int>:··<something>
```
(where `·` are spaces), then `<something>` would end up being interpreted as a path and `<int>` as a line number, even if they were really communicating something different.

But yes, we _could_ actually replace `line[:-1]` with `line.rstrip()` which would also strip other trailing whitespace in addition to the newline. I don't see any benefit in ever keeping trailing spaces or tabs in the description.

And I now realize that leading whitespace is currently actually interpreted as part of the file name! So maybe that should be fixed as well. Since `.py` file name can never start with a space, I think we can safely ignore such lines.

So thanks, I'll make these changes!

Reply:
Changes done. While I was at it, I made linter output parsing still stricter. I think this is a good way to avoid unexpected mishaps in edge cases.